docs/_plugins/scrollable_code_highlight.rb

```diff
@@ -0,0 +1,11 @@
+module Kramdown
```

Comment:
Do we need to explicitly load this anywhere so it can take effect? I'd expect a `_config.yml` change listing this file. Or does it get magically loaded somehow?

Reply:
Anything in `_plugins` is magically loaded by Jekyll, and this will override the module method from Kramdown referenced indirectly through Jekyll compilation.

I adapted this approach from similar plugins we use on some other sites like with how we [add design system classes to Markdown links in the brochure site](https://github.com/GSA-TTS/identity-site/blob/main/_plugins/content_typography.rb).